psi4/src/psi4/fisapt/fisapt.h

```diff
@@ -33,6 +33,8 @@
 #include "psi4/libmints/basisset.h"
 #include "psi4/libmints/wavefunction.h"
 #include "psi4/libpsi4util/PsiOutStream.h"
+#include "psi4/lib3index/dfhelper.h"
```

Comment:
Can you forward declare the class by adding `class DFHelper::DFHelper` around line 44 instead of including this header?

Reply:
fyi, [this](https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/basisset.h#L51) is what a forward declaration looks like.